### PR TITLE
Fix travis, disallow memory-0.14.12

### PR DESCRIPTION
**Pull request by phadej**

### CODE REVIEW

**Comment by alpmestan** [comment]
So the cookbook is not CI-checked at all anymore?

File: README.md [hunk 1]
```diff
@@ -58,3 +58,23 @@ See `CONTRIBUTING.md`
...
+## travis
+
+`.travis.yml` is generated using `make-travis-yml` tool, in
+[multi-ghc-travis](https://github.com/haskell-hvr/multi-ghc-travis) repository.
+
+To regenerate the script use (*note:* atm you need to comment `doc/cookbook/` packages).
+
+```
+runghc ~/Documents/other-haskell/multi-ghc-travis/make_travis_yml_2.hs regenerat
+```
```
Comment (jkarni) [inline README.md]: Hmm, so the idea is that we'd never manually change the `.travis.yml` file?

Comment (phadej) [reply README.md]: yes

**Comment by phadej** [comment]
@alpmestan not right now, I'm working on bringing it back.

**Pull request merged**